addons/trenches/XEH_postInit.sqf

```diff
@@ -23,4 +23,14 @@ GVAR(digDirection) = 0;
 // handle waking up dragged unit and falling unconscious while dragging
 ["ace_unconscious", {_this call FUNC(handleUnconscious)}] call CBA_fnc_addEventHandler;
 
+// Add zeus evh to handle texture on trench move
+(getAssignedCuratorLogic player) addEventHandler ["CuratorObjectEdited", {
```

Comment:
Guessing this won't work properly at all if Zeus logic get's assigned after postInit.

Reply:
Yeah, that’s right but I don’t see a way to detect when a player has become curator.

Reply:
https://community.bistudio.com/wiki/Arma_3:_Event_Handlers#CuratorObjectEdited
The event is local/local, so you could probably just add them to all curators

Reply:
Is it possible to do this EH with XEH? Then it would work all the time however this EH might has to be attached to the curator unit.

Reply:
XEH would at least allow support when somebody gets Zeus while the mission is running already.

Reply:
Ok there is no XEH support for curator events.

Reply:
```
["ModuleCurator_F", "init", {
    params ["_logic"];
    _logic addEventHandler ["CuratorObjectEdited", {
    ...
    }];
}] call CBA_fnc_addClassEventHandler;
```

Reply:
I added an InitPost xeh for curator modules that adds this evh. Have no possibility for testing currently but I assume that this could work.